api/transport/request.go

```diff
@@ -31,6 +31,23 @@ import (
 
 // Request is the low level request representation.
 type Request struct {
+	// ID is a unique identifier for a request/response pair. This MAY be a
+	// trace ID or UUID.
```

Comment:
trace ID may not be possible, because we don't have an opentracing interface to get that ID.